package.json

```diff
@@ -39,9 +39,9 @@
   "main": "lib/index.js",
   "types": "types/index.d.ts",
   "dependencies": {
-    "@react-native-community/art": "^1.2.0",
     "prop-types": "^15.7.2",
-    "qr.js": "0.0.0"
+    "qr.js": "0.0.0",
+    "react-native-svg": "^12.1.1"
```

Comment:
Wouldn't it be better to make `react-native-svg` a peer dependency of this package? From my understanding being a direct dependency it would also get installed (unnecessarily) when on the web.

Reply:
Agree